src/sources/docker_logs.rs

```diff
@@ -4,9 +4,10 @@ use crate::{
     event::merge_state::LogEventMergeState,
     event::{self, Event, LogEvent, Value},
     internal_events::{
-        DockerCommunicationError, DockerContainerEventReceived, DockerContainerMetadataFetchFailed,
-        DockerContainerUnwatch, DockerContainerWatch, DockerEventReceived,
-        DockerLoggingDriverUnsupported, DockerTimestampParseFailed,
+        DockerCommunicationError, DockerContainerMetadataFetchFailed,
+        DockerLoggingDriverUnsupported, DockerLogsContainerEventReceived,
```

Comment:
Should we also rename `DockerCommunicationError`, `DockerContainerMetadataFetchFailed`, `DockerLoggingDriverUnsupported`?